src/Service.Tests/DatabaseSchema-MySql.sql

```diff
@@ -221,6 +222,20 @@ CREATE TABLE books_sold (
   last_sold_on_date DATETIME GENERATED ALWAYS AS (last_sold_on) STORED
 );
 
+CREATE TABLE default_with_function_table
+(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    user_value INT,
+    `current_date` TIMESTAMP DEFAULT (CURRENT_DATE) NOT NULL,
+    `current_timestamp` TIMESTAMP DEFAULT (NOW()) NOT NULL,
```

Comment:
Why do we need to enclose current_date and current_timestamp columns within `` and while its not required for other column names? Are these two reserved names?